draft-secure-credential-transfer.md

```diff
@@ -211,6 +213,30 @@ Sender device may terminate the secure credential transfer by deleting the mailb
 ~~~
 {: #stateful-flow-image title="Sample stateful workflow"}
 
+## Credential Vertical in share url
```

Comment:
I wasn't entirely sure on the best place for this section. What does everyone think?

Reply:
It seems we have definitions of parts of the URL scattered across the spec, so I would say adding a dedicated section to define the URL and it's parts, including if they are required\optional, would be better.

Reply:
I noticed that too, let me combine the various URL sections.

Reply:
I added a more general share url section, but I wasn't how much detail to go into for each field. Do you think it would be worth recapping a description for each field in this new section?